### ISSUE TITLE
Revive Rudimentary "Chord-Pulling" Capability

### ISSUE THREAD

**Issue by crantila** [ELVIS-Project/vis-framework#258]
Since using music21's `reduceChords()` method (as proposed in issue #210) is now antithetical to the VIS Framework's query-design strategy, we must develop our own "chord-finding" algorithms.

The first step is to revive the `ChordParser` as found in the vis9 series (refer to [this](https://github.com/ELVIS-Project/vis/blob/vis9d/controllers/experimenter.py#L1619)).

For the under-informed: the existing `ChordParser` is a rather naïve and preliminary attempt to "pull" "chords" "out of" "piano textures." Basically, it takes all the simultaneous notes at a moment, checks if it's a chord, and if not, it adds the next set of simultaneous-notes-at-a-moment, up to an arbitrary limit of additional simultaneous-notes-at-moments, until it either finds a nice chord and adds it for the whole duration span, or gets to the limit and gives up, adding only the first set of simultaneous notes at a moment as a not-nice chord.

It's worse than it should be, but better than it could be.

**Comment by alexandermorgan** [comment]
@mborsodi could you look into implementing some music21 function to come up with a chordal analysis indexer?

**Comment by musicus** [comment]
@alexandermorgan didn't we specifically assign Claire to this task?

**Comment by crantila** [comment]
... wow... you totally unearthed the 2013-era issue!

I would be happy to talk this over with whoever's doing it, since I've made several attempts, most of whihc aren't public, and none of which are compatible with VIS in May 2016.

**Comment by alexandermorgan** [comment]
Yes Reiner, you're right, this was assigned to Claire. Thanks for the reminder!

**Comment by crantila** [comment]
Is there still interest in this task from ELVIS? If not, someone can close it.

**Comment by musicus** [comment]
There is (didn't mean to un-assign you).
